backend/app/services/collections/create_collection.py

```
@@ -71,142 +73,248 @@ def start_job(
+        return None
+
+
 def execute_job(
     request: dict,
     project_id: int,
     organization_id: int,
-    payload: dict,
     task_id: str,
     job_id: str,
+    with_assistant: bool,
     task_instance,
```

Ruff (0.14.3)

Unused function argument: `task_instance`

(ARG001)